src/DynamoCore/DynamoCore.csproj

```diff
@@ -380,43 +379,40 @@ limitations under the License.
   <ItemGroup>
     <Resource Include="BuiltInAndOperators\BuiltIn.Migrations.xml" />
   </ItemGroup>
-  <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="BeforeBuild">
     <ItemGroup>
-      <ExternProtoGeometry Include="$(SolutionDir)..\extern\ProtoGeometry\*" />
-      <ExternProtoGeometryXml Include="$(SolutionDir)..\extern\ProtoGeometry\ProtoGeometry.XML" />
-      <ExternProtoGeometryUICulture Include="$(SolutionDir)..\extern\ProtoGeometry\$(UICulture)\*" />
-      <ExternProtoGeometryLibGLocale Include="$(SolutionDir)..\extern\ProtoGeometry\libg_locale\**\*.*" />
-      <ExternLibG224 Include="$(SolutionDir)..\extern\LibG_224_0_1\*" />
-      <ExternLibG2244 Include="$(SolutionDir)..\extern\LibG_224_4_0\*" />
-      <ExternLibG225 Include="$(SolutionDir)..\extern\LibG_225_0_0\*" />
-      <ExternLibG226 Include="$(SolutionDir)..\extern\LibG_226_0_0\*" />
-      <ExternLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
+      <LibGInterface Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_226_0_0*\tools\net48\LibG.Interface.dll" />
+      <LibGProtoGeometryXml Include="$(solutiondir)\packages\dynamovisualprogramming.libg_226_0_0*\lib\net48\protogeometry.xml" />
+      <LibGProtoGeometryUICulture Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_226_0_0*\tools\net48\$(UICulture)\*" />
+      <LibGProtoGeometryLibGLocale Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_226_0_0.2.9.0.2375\tools\net48\libg_locale\**\*.*" />
```

Comment:
> That it doesn't matter which version of LibG_226_0_0 are you using that is pasted on the name of the folder, it will take that file, so we don't need to update that line every time we update the LibG nuget reference

In that case, why not use the `*` here as well?

Reply:
Because on line #411 we are using a %(RecursiveDir) that doesn't work well with the * on that part

Reply:
So it looks like we'll have to update this csproj for every single nuget package update? I think that'll be too much. Is there a way to save us from doing that?

Reply:
Using only the csproj element as it is, no, but now I'm thinking that we can use a similar approach of what we did on LibG where we are updating nuget.exe with powershell, pretty sure that doesn't have issues with the * on that part and it can be recursive, I'll do that change.

Reply:
Other option could be to change all the projects to use packagareference instead of package.config, and with that, we can create a path alias (https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#generatepathproperty)